app/models/orcid_work.rb

```diff
@@ -0,0 +1,61 @@
+class OrcidWork < OrcidResource
+  def to_json
+    external_ids = %i[isbn issn doi url]
+    url = publication.open_access_url || publication.url
+
+    work = {
+      title: {
+        title: publication.title,
+        subtitle: publication.secondary_title,
+      },
+      "journal-title": publication.journal_title,
+      "short-description": publication.abstract,
+      type: 'journal-article',
+      "publication-date": {
+        year: publication.published_on.year,
+        month: publication.published_on.month,
+        day: publication.published_on.day
+      },
+      url: url
+    }
+
+    external_ids.each do |type|
```

Comment:
You'll have to explain the orcid "external-id" concept to me...

Reply:
Oh, never mind. I think I see what's going on. 

It occurs to me that a user of our app won't really be able to see some of the data that we're going to be adding to their orcid works. Some of these bits of data are an example of that.

Knowing that some of this data isn't super clean (especially in the case of things that the user has added to Activity Insight), I wonder if users might be surprised by some of the information that ends up in orcid. That might be a whole additional workflow issue that we may need to consider.

I'm thinking that it might be good to give the user some kind of preview of all of the fields/values that are going to be added to orcid for each work before they confirm that they want to do it. Or at the very least, maybe we need to give users the ability to delete things that they've added to orcid if they see that some of the data isn't quite right after the fact. Actually handling correction of any bad data is a whole other issue, but it would probably be the responsibility of a metadata admin.